src/ucm/mmap/install.c

```diff
@@ -231,11 +231,10 @@ static void ucm_mmap_event_report_missing(int expected, int actual,
 
     buf   = buf_p = ucs_alloca(UCM_MMAP_REPORT_BUF_LEN);
     end_p = buf_p + UCM_MMAP_REPORT_BUF_LEN;
-    ucs_for_each_bit(idx, expected & ~actual) {
-        if (idx > ucs_log2(UCM_EVENT_VM_UNMAPPED)) { /* for coverity */
-            break;
-        }
 
+    ucs_for_each_bit(idx,
+                     expected & ~actual & ((UCM_EVENT_VM_UNMAPPED << 1) - 1)) {
```

Comment:
maybe 
int missing_events;
missing_events = expected & ~actual & UCS_MASK(UCM_EVENT_VM_UNMAPPED)
BTW it should be `1<< UCM_EVENT_VM_UNMAPPED`, not `UCM_EVENT_VM_UNMAPPED << 1`

Reply:
added variable

> BTW it should be `1<< UCM_EVENT_VM_UNMAPPED`, not `UCM_EVENT_VM_UNMAPPED << 1`

nop, ```UCM_EVENT_VM_UNMAPPED == UCS_BIT(17)```, and ```1<< UCM_EVENT_VM_UNMAPPED``` equals ```1 << UCS_BIT(17)``` - it is out of ```uint64_t```